x-pack/solutions/security/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -151,9 +155,17 @@ export class CspPlugin
           'packagePolicyUpdate',
           async (
             packagePolicy: UpdatePackagePolicy,
-            soClient: SavedObjectsClientContract
+            soClient: SavedObjectsClientContract,
+            esClient: ElasticsearchClient
           ): Promise<UpdatePackagePolicy> => {
             if (isCspPackage(packagePolicy.package?.name)) {
+              const isIntegrationVersionIncludesTransformAsset =
+                parseInt(packagePolicy.package!.version.split('.')[0], 10) >= 2;
```

Comment:
nit: we usually use `semver` to work with versions, a bit more straightforward and hopefully error prone